pennylane_lightning/core/bindings/BindingsMPI_nb.hpp

```diff
@@ -88,13 +96,11 @@ template <class StateVectorT> void registerObservablesMPI(nb::module_ &m) {
         typename StateVectorT::PrecisionT; // Statevector's precision.
     using ComplexT =
         typename StateVectorT::ComplexT; // Statevector's complex type.
-    using ParamT = PrecisionT;           // Parameter's data precision
 
     const std::string bitsize =
         std::is_same_v<PrecisionT, float> ? "64" : "128";
 
-    using arr_c = nb::ndarray<std::complex<ParamT>, nb::c_contig>;
-    using arr_r = nb::ndarray<ParamT, nb::c_contig>;
+    using arr_c = nb::ndarray<std::complex<PrecisionT>, nb::c_contig>;
```

Comment:
Should we call this something like `ArrayT` to stick with the naming convention for type names?